modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java

```diff
@@ -68,6 +69,34 @@ public KBSelectParentDisplayContext(
 		_initParentData();
 	}
 
+	public String getContentSelectionTitle() throws PortalException {
+		long kbFolderClassNameId = PortalUtil.getClassNameId(
+			KBFolderConstants.getClassName());
+		long contentSelectionId = getParentResourcePrimKey();
```

Comment:
Can you rename `contentSelectionId` to `resourcePrimKey`?

Reply:
Even better; as this is used only once, can you inline it?

Reply:
I've edited it. Thank you!